static/bidder-info/tappx.yaml

```diff
@@ -5,3 +5,7 @@ capabilities:
     mediaTypes:
       - banner
       - video
+  site:
+    mediaTypes:
+      - banner
+      - video
```

Comment:
Please add `adapters/tappx/tappxtest/exemplary` files for site. The current ones are for app.

Reply:
This is done